app/mirage/fixtures/settings.js

```diff
@@ -3,7 +3,7 @@ export default [
     {
         created_at: '2015-09-11T09:44:30.805Z',
         created_by: 1,
-        id: 1,
+        id: 7,
```

Comment:
Just me being 🐷... Didn't know which ID to use for `amp`, so I updated them regarding the ID from our API response.